test/unit/parse-cookie-string.test.ts

```diff
@@ -1,39 +0,0 @@
-/* eslint-env jest */
-import { parseCookieString } from 'next/dist/server/web/spec-extension/cookies/serialize'
```

Comment:
Moved to https://github.com/vercel/edge-runtime/blob/441f6be0f0354cc1f4ce5d2dbf7da013de914989/packages/cookies/test/request-cookies.test.ts#L121